lib/services/notifications.js

```diff
@@ -31,9 +31,13 @@ function callWebhook(event, url, data) {
     options.body = data;
   }
 
-  return rest.fetch(url, options).then(function (response) {
-    log('info', `called webhook ${_.pick(response, ['status', 'statusText'])}`);
-  });
+  return rest.fetch(url, options)
+    .then(function (response) {
+      log('info', `called webhook ${_.pick(response, ['status', 'statusText'])}`);
```

Comment:
This doesn't relate to these changes specifically, but I have noticed that `node-fetch` doesn't terminate http agents (i.e. free up sockets) immediately unless responses are actually read (`res.buffer()`, `res.text()`, `res.json()`).

Reply:
Oh, that's right. Let me add a thing for that.